.github/workflows/lint.yml

```diff
@@ -34,15 +34,16 @@ jobs:
         go-version: 1.17.8
       id: go
 
-    - name: install crlfmt
+    - name: install gofumpt
       working-directory: src/go/rpk
       run: |
-        go mod download github.com/cockroachdb/crlfmt
-        go install github.com/cockroachdb/crlfmt
+        go mod download mvdan.cc/gofumpt@latest
+        go install mvdan.cc/gofumpt@latest
```

Comment:
Why latest? Also I don't think the `go mod download` line is necessary.

Reply:
Fixed with v0.3.0